lib/routes/account.js

```diff
@@ -425,6 +432,15 @@ module.exports = function (
             .then(
               function (result) {
                 device = result
+                if (operation === 'createDevice') {
+                  log.event('device', {
+                    action: 'create',
```

Comment:
> any preference?

I'm fine with whatever, but I would support an effort to namespace events in general.